users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -233,10 +233,17 @@ def run_scf_baseline():
                 lr_args=lr_args,
                 report_args={"batch_size": "2x5k"},
             ),
+            "bs2x5k_scf_baseline_preemphasis97_wn": dict(
+                returnn_args=returnn_args,
+                feature_args= {**feature_args, "preemphasis": 0.97, "wave_norm": False},
+                lr_args=lr_args,
+                report_args={"batch_size": "2x5k", "preemphasis": "0.97", "wave_norm": "False"},
```

Comment:
preemphasis and wave norm are automatically added to the report in `baseline_args.py` .